### DISCUSSION TITLE
Python / Node.js

### DISCUSSION THREAD

**Discussion by eaglerockdude** [codefori/vscode-ibmi#1822] [General]
I started using IBMi to try out the RPG features, but lately I have a need to write programs/scripts to help with IFS based text files.  Mostly Python, and also Node.js.  Oh yes...I wanted to use the VS Code editor is mostly why.

Is there a forum or thread or is anyone else doing that...is this the right place?

**Comment by beckhamrryyaann** [comment]
People are definitely doing this.  You can do local development on your pc to code your Node/python apps, meaning you write the source code local on your machine using VSCode.   That way you can test/debug your app locally. 

When you are ready to move your app to the IBMi server.  You would connect to your IBMi server  using Code4i and you would set a deploy location from the IFS Browser.  This would be where you want your app to run from in the IFS.   Then all you would need to do is deploy your source code in Code4i.  This moves your source to the IFS.  Liam has already made a video about this as well on his youtube channel.

This method also allows you easily store in code in a GIT repository and have your project under source control.
